ui/packages/consul-ui/docs/hds.mdx

```diff
@@ -57,44 +57,40 @@ Because we are working with a customized tailwind configuration it is easy to co
 All components from Hashicorp Design System are available for you to use. Here's an example that shows how to implement a navigation bar with HDS and Tailwind in combination.
 
 ```hbs preview-template
-<nav class="h-16 w-full bg-hds-foreground-strong flex items-center justify-between px-4 hds-font-weight-medium">
-  <ul class="flex items-center">
+<nav
+  class='h-16 w-full bg-[color:var(--token-color-palette-neutral-700)] flex items-center justify-between px-4 hds-font-weight-medium'
```

Comment:
We probably want to switch to a class name for the background here. I wonder if we should be using a custom utility class here, and how to name said utility class, as no dark surface color with a `hds-`-prefix exists yet: 

We could leave out the prefix altogether
```css
.surface-nav {
  background: var(--token-color-palette-neutral-700);
}
```
or create our own prefix to indicate that we tried using HDS but couldn't as a utility that does what we want didn't exist yet:

```css
.consul-surface-nav {
  background: var(--token-color-palette-neutral-700);
}
```

Reply:
@LevelbossMike I haven't looked at the Consul UI codebase, but I would expect that there is only one instance of this UI element (the page header) in it, and it's expressed via a component (eg. `Header`) or directly in the page template. This means (under my assumption) that probably defining a utility class only for that single instance of background color is a bit of an overkill. Probably it would be (in principle) more "correct" to define a local class (eg. `.consul-header`), assign the background color as suggested above (`background: var(--token-color-palette-neutral-700);`) and then apply that class to the `<nav>` element (the one suggested above).

My suggestion is (since this is not actual code, but a README) to change the example, or use it but assign the "local" class name and explain how and why you have used it this way (consider it an example of what _not_ to do :) ). Unless the approach is to use only tailwind + hds classes in the handlebars templates, and never create specific local/custom class names for the Consul UI components/elements, in which case... it's going to be challenging and probably you have to rely on tailwind helpers like `bg-[color:var(--token-color-palette-neutral-700)]`.

Reply:
Thanks for the input! I discussed this with @WenInCode and for consistency’s sake, we'll take the custom utilty approach here. With Tailwind, this reads nicer, and we will only have one place where this custom surface class would be defined. 

Example:

```hbs
<nav class="flex items-center space-x-2 consul-surface-brand">
```
vs.

```hbs
<nav class="flex items-center space-x-2 consul-header" />
```